server/container/guice/cassandra-guice/src/main/java/org/apache/james/modules/webadmin/InconsistencySolvingRoutesModule.java

```diff
@@ -17,7 +17,7 @@
  * under the License.                                           *
  ****************************************************************/
 
-package org.apache.james.modules.server;
+package org.apache.james.modules.webadmin;
```

Comment:
why did you put webadmin stuff in a guice project?

Reply:
I did put some guice registrations in a guice project. What's wrong?

Reply:
That's ok